src/Symfony/Component/Lock/Store/MongoDbStore.php

```diff
@@ -315,23 +317,15 @@ private function isDuplicateKeyException(WriteException $e): bool
         return 11000 === $code;
     }
 
-    private function getDatabaseVersion(): string
+    private function getReadPreference(): ReadPreference
```

Comment:
This appears to be called exactly once in `exists()`. ReadPreference is a very light value object, so I'd propose just removing this method (and cached class property) in favor of constructing it inline.

Reply:
removed